client/js/_author/components/assessments/question_types/multiple_choice.jsx

```diff
@@ -72,7 +76,9 @@ export default class MultipleChoice extends React.Component {
           ))
         }
         {
-          this.props.isActive ? <Add updateChoice={() => this.props.updateChoice(id)} /> : null
+          this.props.isActive ? <Add
+            updateChoice={() => this.addNewChoice()}
```

Comment:
you already have id defined above, pass it in so you don't have to redeclare it in function